package.json

```diff
@@ -22,7 +22,7 @@
     "start:prod": "NODE_ENV=development ENVIRONMENT=prod webpack serve --config config/dev.webpack.config.js",
     "start:prod:beta": "NODE_ENV=development BETA=true ENVIRONMENT=prod webpack serve --config config/dev.webpack.config.js",
     "local": "NODE_ENV=development webpack serve --config config/dev.local.webpack.config.js",
-    "test": "TZ=UTC jest --verbose",
+    "test": "TZ=UTC jest --verbose --no-cache",
```

Comment:
We're finding that this is not needed on every repo. If your tests are too slow you can try to remove this and see if it works.

Reply:
Thank you for the review and inputs @BlakeHolifield I will keep it in mind and do changes if needed! 💯